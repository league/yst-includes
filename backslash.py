print "Hello\\n\n"
print "What about $blah$"
# $$
