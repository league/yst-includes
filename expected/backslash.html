<style>
.prompt { color: green; }
.cmd { color: blue; }
</style>

<pre include="backslash.py"><code>print &quot;Hello\\n\n&quot;
print &quot;What about $blah$&quot;
# $$</code></pre>
